client/components/jetpack/automattic-byline-logo/README.md

```diff
@@ -0,0 +1,16 @@
+# Automattic Byline Logo
```

Comment:
What's the rationale for adding these components to `client/components/jetpack` instead of `packages/components`?

Reply:
The by-line logo is 'An Automattic Airline' which IMO is Jetpack 😄